internal/inventory/azurefetcher/fetcher_storage.go

```diff
@@ -104,17 +104,13 @@ func (f *storageFetcher) fetch(ctx context.Context, storageAccounts []azurelib.A
 	for _, item := range azureAssets {
 		assetChan <- inventory.NewAssetEvent(
 			classification,
-			[]string{item.Id},
+			item.Id,
 			item.DisplayName,
 			inventory.WithRawAsset(item),
-			inventory.WithCloud(inventory.AssetCloud{
-				Provider: inventory.AzureCloudProvider,
-				Account: inventory.AssetCloudAccount{
-					Id: item.TenantId,
-				},
-				Service: &inventory.AssetCloudService{
-					Name: "Azure",
-				},
+			inventory.WithCloud(inventory.Cloud{
+				Provider:    inventory.AzureCloudProvider,
+				AccountID:   item.TenantId,
+				ServiceName: "Azure",
```

Comment:
maybe use `inventory.AssetClassification` to make a more specific `ServiceName`

Reply:
There is no given guidance on what to add here. I might mimic AWS for consistency when I'm going over ECS fields in the next PR.